src/buildHTML.js

```diff
@@ -150,11 +150,6 @@ export const buildExpression = function(expression, options, isRealGroup,
     for (let i = 0; i < groups.length; i++) {
         if (groups[i].value === "\u0338") {
             groups[i].style.position = "absolute";
```

Comment:
Is any of this loop still needed?  Can you test without it?

Reply:
It is still necessary.  The `position = "absolute"` causes the `\u0338` to overlay the the following group/atom.

Reply:
Is this equivalent to `\rlap`? An alternative would be to macro `\not` to `\rlap{\not@}` and rename the symbol to `\not@`.  I'm not quite sure what the intended behavior is, as LaTeX's source code is pretty opaque:
```latex
\DeclareMathSymbol{\not}{\mathrel}{symbols}{"36}
```

Reply:
I'll give the `\rlap` approach a try.  I'm not sure what to make sure of that chunk of LaTeX's source though.